src/App.js

```diff
@@ -126,6 +127,7 @@ const App = ({ locale, location, dependencies: { appSessionRef, sessionManager }
                 exact
                 component={ReportsPartialsCampaigns}
               />
+              <PrivateRoute path={['/invoices-list']} exact component={InvoicesList} />
```

Comment:
Maybe `billing/invoices`?

Reply:
Well, this URL can be modified in the future, so, it is not so important, but I am starting to think about the UI, there will probably be a section for "billing" and inside that section a subsection for "invoices", for that reason I like a hierarchical URL.

Reply:
for me is ok if you want that I change the url.

Reply:
@lucianaDilizio what do you think?

Reply:
I don't have a strong opinion about that. I think that `billing/invoices` is ok. 👍